Controller/CategoryAdminController.php

```diff
@@ -99,7 +98,7 @@ public function treeAction(Request $request)
         $datagrid = $this->admin->getDatagrid();
 
         if ($this->admin->getPersistentParameter('context')) {
-            $datagrid->setValue('context', ChoiceType::TYPE_EQUAL, $this->admin->getPersistentParameter('context'));
+            $datagrid->setValue('context', null, $this->admin->getPersistentParameter('context'));
```

Comment:
I still don't understand… you say you are fixing a missing type, and the diff shows you are in fact removing it. What's the deal ? 😕